db/migrations/02_create_users_table.sql

```diff
@@ -0,0 +1 @@
+CREATE TABLE users (id SERIAL PRIMARY KEY, email VARCHAR(60), password VARCHAR(140)), username VARCHAR(60);
```

Comment:
There's a bracket in the wrong place here which prevents you from setting up database correctly. 
But I managed it and the site looks great!